app/views/events/show.html.erb

```diff
@@ -57,6 +57,10 @@
         </p>
       </div>
     </div>
+
+
+    <%= render :partial => 'extended_metadata/extended_metadata_attribute_values', :locals => {:resource => @event} %>
```

Comment:
use the format `partial:` rather than `:partial=>` , and the same for other keys